packages/zk-transfer-web/src/components/Header.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import { shortenAddress } from '../helpers/account';
 
-const Header = () => {
+const Header = ({ thisAddress }: { thisAddress: string }) => {
```

Comment:
is there a difference between this and
```
({ thisAddress: string}) => {
```
?

Reply:
Yeah. For this `({ thisAddress: string })`, the `string` becomes an alias of `thisAddress` variable. 
While `({ thisAddress } : { thisAddress: string })` setting the types for the Component props.

Reply:
oh i see, it's a Typescript thing. That makes sense, and I like your way with general `HeaderProps` better.